src/esp/assets/ResourceManager.cpp

```diff
@@ -149,8 +149,8 @@ bool ResourceManager::loadPTexMeshData(const AssetInfo& info,
 
   return true;
 #else
-  LOG(ERROR)
-      << "PTex support not enabled. Define BUILD_PTEX_SUPPORT when building.";
+  LOG(ERROR) << "PTex support not enabled. Define ESP_WITH_PTEX_SUPPORT when "
```

Comment:
This naming inconsistency between `BUILD_*` and `WITH_*` might be a problem for users -- could the CMake `option()` be renamed to `WITH_*` as well? Then the error message would be clearer as

```suggestion
  LOG(ERROR) << "PTex support not enabled. Enable the WITH_PTEX_SUPPORT CMake option when "
```

Reply:
I can change things to `BUILD_*` to be consistent with the current cmake options

Reply:
Or that, I don't have a preference. In Magnum the difference between `WITH` and `BUILD` is kinda blurry also, so ¯\\\_(ツ)_/¯